modal_images/gemma.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import annotations
+
+from modal import Image, Secret, Stub
+
+# Define the model names for LLaMA-2, Mistral, and GPT-2
+model_names = {
+    "llamatiny": "TinyLlama/TinyLlama-1.1B-intermediate-step-1431k-3T",
+    "llama": "meta-llama/Llama-2-13b-hf",
+    "gpt2": "gpt2",
+    "mistral": "mistralai/Mistral-7B-v0.1",
+    "gptj": "EleutherAI/gpt-j-6B",
+    "gemma2": "google/gemma-2b",
+    "gemma7": "google/gemma-7b",
+}
+
+
+def download_model():
+    """
+    Download the model and move the cache
+    """
+    import os
+
+    token = os.environ["HUGGINGFACE_TOKEN"].strip()
+    from huggingface_hub import snapshot_download
+    from transformers.utils import move_cache
+
+    snapshot_download(model_names["gemma7"], token=token, revision="main")
+    snapshot_download(model_names["gemma2"], token=token, revision="main")
+    move_cache()
```

Comment:
download the model images into the docker container so when you run scripts its fast to load